microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -129,8 +128,8 @@ The following shortcuts are always turned off in WebView2.  An asterisk (`*`) in
 | Select Tab (1 - 8) | `Ctrl`+`(1-8)` |
 | Show Favorites Bar `*` | `Ctrl`+`Shift`+`B` |
 | Help | `F1` |
-| Focus Next Pane `*` | `F6` |
-| Focus Previous Pane `*` | `Shift`+`F6` |
+| Focus Next Pane | `F6` |
+| Focus Previous Pane | `Shift`+`F6` |
```

Comment:
@tofuandeve @jasonstephen15 - Is Shift+F6 supported now?  Not mentioned in my work item & linked work item.  I would assume so; F6 & Shift+F6 are always "linked" as complementary in any UI.

Reply:
Yes both F6 and Shift+F6 are partially supported now in webview2 using windowed hosting mode